scripts/tokenomics-emulation/constants.ts

```diff
@@ -0,0 +1,23 @@
+const config = {
```

Comment:
Rename the file so it's clear which constants. For instance, `network-custom-types`.